eyeshade/controllers/owners.js

```diff
@@ -44,14 +44,15 @@ v1.bulk = {
     return async (request, reply) => {
       const authorizer = request.payload.authorizer
       const providers = request.payload.providers || []
-      let extras
+      const channels = []
+      let info = {}
 
-      if (authorizer.ownerEmail || authorizer.ownerName) {
-        extras = underscore.omit(authorizer, [ 'owner' ])
-        providers.forEach((provider) => { underscore.extend(provider, extras) })
-      }
+      if (authorizer.ownerEmail || authorizer.ownerName) info = { email: authorizer.ownerEmail, name: authorizer.ownerName }
```

Comment:
should this be `&&`?

Reply:
it should be `||` if either is set, use the either/both values. if either, one will be `undefined` which worksin this case.